Example-iOS/DemoStrings.swift

```diff
@@ -386,26 +394,22 @@ enum DemoStrings {
                 .style("zh-TW", StringStyle(.speakingLanguage("zh-TW"))),
                 ]))
 
-        let stringsArrays = [
-            [
+        let strings = [
                 "<speak-punctuation>With VoiceOver<punctuation>,</punctuation> this string<punctuation>,</punctuation> well<punctuation>,</punctuation> reads its<punctuation>…</punctuation>punctuation<punctuation>!</punctuation></speak-punctuation>",
-                ],
-            [
                 "<high-pitch>Helium is less dense than air, making my voice higher. </high-pitch><low-pitch>And with sulfur hexafluoride, my voice gets really low, and somehow, I’m still funny. Ha ha ha.</low-pitch>",
-                ],
-            [
-                "<el-GR><emoji>🇬🇷</emoji> Εδώ είναι ο σωστός τρόπος για να προφέρει Ελληνικά.</el-GR>",
-                "<en-US>\n<emoji>🇺🇸</emoji> Here is the correct way to pronounce American English.</en-US>",
-                "<es-DO>\n<emoji>🇩🇴</emoji> Aquí está la manera correcta de pronunciar el español.</es-DO>",
-                "<fr-FR>\n<emoji>🇫🇷</emoji> Voici comment prononcer correctement le français.</fr-FR>",
-                "<pt-PT>\n<emoji>🇵🇹</emoji> Aqui está a maneira correta de pronunciar português.</pt-PT>",
-                "<ru-RU>\n<emoji>🇷🇺</emoji> Вот как правильно произносить русский.</ru-RU>",
-                "<uk-UA>\n<emoji>🇺🇦</emoji> Ось як правильно вимовляти Український.</uk-UA>",
-                "<zh-TW>\n<emoji>🇹🇼</emoji> 这里是正确的做法发音中文.</zh-TW>",
-                ],
+                """
+                <el-GR><emoji>🇬🇷</emoji> Εδώ είναι ο σωστός τρόπος για να προφέρει Ελληνικά.</el-GR>
+                <en-US><emoji>🇺🇸</emoji> Here is the correct way to pronounce American English.</en-US>
+                <es-DO><emoji>🇩🇴</emoji> Aquí está la manera correcta de pronunciar el español.</es-DO>
+                <fr-FR><emoji>🇫🇷</emoji> Voici comment prononcer correctement le français.</fr-FR>
+                <pt-PT><emoji>🇵🇹</emoji> Aqui está a maneira correta de pronunciar português.</pt-PT>
+                <ru-RU><emoji>🇷🇺</emoji> Вот как правильно произносить русский.</ru-RU>
+                <uk-UA><emoji>🇺🇦</emoji> Ось як правильно вимовляти Український.</uk-UA>
+                <zh-TW><emoji>🇹🇼</emoji> 这里是正确的做法发音中文.</zh-TW>
+                """,
```

Comment:
This adds newlines where you previously had none, so depending on whether you want them or not, you can end lines with a `\` to remove the newline 🙂

Reply:
Good catch. It turns out, the VoiceOver pronunciation hinting system is pretty broken, and if I have the spaces and new lines set up any way other than what was here before, with a newline before `<emoji>` and a space after `</emoji>`, VoiceOver says "Space" in the default Siri voice after reading each sentence. Since multiline string literals actually make the correct arrangement _harder_ to read, I'm going to revert this section.

Reply:
👍